### PR TITLE
acm: add release-2.1 for work and registration

### PR DESCRIPTION
**Pull request by ray-harris**

### CODE REVIEW

**Comment by ray-harris** [comment]
/assign @qiujian16
/assign @pmorie 

Can one of you review please? I'm not concerned about the one e2e rehearsal that's failing.

File: ci-operator/jobs/open-cluster-management/work/open-cluster-management-work-release-2.1-postsubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        - -f
+        - /opt/build-harness/Makefile.build-harness-openshift-ci
+        - pipeline-manifest/update
+        command:
+        - make
+        env:
+        - name: COMPONENT_NAME
+          value: work
+        - name: COMPONENT_VERSION
+          value: 0.0.1
```
Comment (qiujian16) [inline ci-operator/jobs/open-cluster-management/work/open-cluster-management-work-release-2.1-postsubmits.yaml:71]: i think we should change this 0.0.2

Comment (ray-harris) [reply ci-operator/jobs/open-cluster-management/work/open-cluster-management-work-release-2.1-postsubmits.yaml]: done

**Comment by qiujian16** [comment]
@ray-harris I think  you would want to squash the commits...

**Pull request closed without merging**